cob_hardware_config/components/sick_s300/sick_s300.yaml

```diff
@@ -1,9 +1,4 @@
-port: /dev/ttyUSB0
 baud: 500000
 scan_duration: 0.025 #no info about that in SICK-docu, but 0.025 is believable and looks good in rviz
 scan_cycle_time: 0.040 #SICK-docu says S300 scans every 40ms
-inverted: false
-scan_id: 7
-frame_id: base_laser_rear_link
 publish_frequency: 12
-scan_intervals: [[-2.0, 2.0]] #[rad] these intervals are included to the scan
```

Comment:
Apart from that, I once heard about a "templatable" yaml approach (https://github.com/ros-industrial/universal_robot/issues/144), but I never got an answer about how to use it...
Does someone know more about it?